exercises/isogram/README.md

```diff
@@ -42,7 +42,8 @@ The [exercism/rust](https://github.com/exercism/rust) repository on GitHub is th
 If you want to know more about Exercism, take a look at the [contribution guide](https://github.com/exercism/docs/blob/master/contributing-to-language-tracks/README.md).
 
 [help-page]: http://exercism.io/languages/rust
-[crates-and-modules]: http://doc.rust-lang.org/stable/book/crates-and-modules.html
+[modules]: https://doc.rust-lang.org/book/second-edition/ch07-00-modules.html
```

Comment:
It looks like the links are expressed both in footnote form and inline. I'm pretty sure this is redundant: if we have an inline link of the form `[link_text](https://...)`, we don't need to include a footnote with the link's value. Contrariwise, if we want to use the footnote style, then the place where the link is actually used should look like `[Modules][modules]`.